.env

```diff
@@ -0,0 +1 @@
+# This file is here for dynaconf to set this as root level directory
```

Comment:
bad wording
```
You can put in the root of your application a file called .env to automatically load some env variables by 

settings = Dynaconf(load_dotenv=True)
```

Reply:
I don't understand. Your proposed wording has completely different meaning than the current one.

Clearly, the intent behind this file is to tell dynaconf where root of the project is. So relative paths used in `LazySettings()` instantiation are resolved relative to directory with this file. It's not there as yet another source of settings.